README.md

```diff
@@ -45,3 +45,20 @@ Which allows for at least some automation. Further info can be found in the
 The sample players as well as the "automatic referee" are implemented in
 Python, which should allow for easily updating the code to match the rules and
 avoid any compilation issues.
+
+## Development
+
+We are open to contributions! Have a look at our [issues](https://github.com/RoboCupJuniorTC/rcj-soccer-sim/issues).
+Before you make a pull request, make sure the code is formatted
+with `black` and `isort`, and `flake8` issues are fixed.
+
+To do so, follow these steps:
+
+1. Create virtualenv `virtualenv -p python3 venv`
```

Comment:
@Adman should this use `venv` perhaps? (As in https://docs.python.org/3/library/venv.html)

Reply:
Yeah, let's use that.